drivers/dma/dma_nxp_sdma.c

```diff
@@ -20,6 +20,15 @@ LOG_MODULE_REGISTER(nxp_sdma);
 AT_NONCACHEABLE_SECTION_ALIGN(static sdma_context_data_t
 			      sdma_contexts[FSL_FEATURE_SDMA_MODULE_CHANNEL], 4);
 
+enum sdma_channel_state {
+	SDMA_CHAN_STATE_INIT = 0,
+	SDMA_CHAN_STATE_CONFIGURED,
+	SDMA_CHAN_STATE_STARTED,
+	SDMA_CHAN_STATE_STOPPED,
+	SDMA_CHAN_STATE_SUSPENDED,
```

Comment:
seems unused?

Reply:
@LaurentiuM1234, added them to match the states in the edma driver. I think prefer to keep them for that reason. Likely use them when add PM support.